src/node_util.cc

```diff
@@ -29,6 +28,7 @@ using v8::Value;
   V(isRegExp, IsRegExp)                                                       \
   V(isSet, IsSet)                                                             \
   V(isSetIterator, IsSetIterator)                                             \
+  V(isSharedArrayBuffer, IsSharedArrayBuffer)                                 \
```

Comment:
Looks like this changes in this file can be removed (it's just moving the line).